docs/UsersGuide/source/BuildingRunningTesting/WE2Etests.rst

```diff
@@ -182,86 +235,23 @@ The final job summary is written by the ``print_WE2E_summary()``; this prints a
-
-   .. code-block::
-
-      ./run_WE2E_tests.py -m=jet -a=gsd-fv3-dev -t=all -q -p 6
-
-   Depending on your machine settings, this can reduce the time it takes to run all experiments substantially. However, it should be used with caution on shared resources (such as HPC login nodes) due to the potential to overwhelm machine resources. 
-
-#. This example will run the single experiment "custom_ESGgrid" on Hera, charging computational resources to the "fv3lam" account. For this example, we submit the suite of tests using the legacy :term:`cron`-based system:
-
-.. note::
-
-   This option is not recommended, as it does not work on some machines and can cause system bottlenecks on others.
-
-   .. code-block::
-
-      ./run_WE2E_tests.py -t=custom_ESGgrid -m=hera -a=fv3lam --use_cron_to_relaunch --cron_relaunch_intvl_mnts=1
-
-The option ``--use_cron_to_relaunch`` means that, rather than calling the ``monitor_jobs()`` function, the ``generate_FV3LAM_wflow()`` function will create a new :term:`cron` job in the user's cron table that will launch the experiment with the workflow launch script (``launch_FV3LAM_wflow.sh``). By default this script is run every 2 minutes, but we have changed that to 1 minute with the ``--cron_relaunch_intvl_mnts=1`` argument. This script will run until the workflow either completes successfully (i.e., all tasks SUCCEEDED) or fails (i.e., at least one task fails). The cron job is then removed from the user's cron table.
-
-
-Checking test status and summary
+Checking Test Status and Summary
 =================================
-By default, ``./run_WE2E_tests.py`` will actively monitor jobs, printing to screen when jobs are complete (either successfully or with a failure), and print a summary file ``WE2E_summary_{datetime.now().strftime("%Y%m%d%H%M%S")}.txt``.
-However, if the user is using the legacy crontab option, or would like to summarize one or more experiments that are either not complete or were not handled by the WE2E test scripts, this status/summary file can be generated manually using ``WE2E_summary.py``.
-In this example, an experiment was generated using the crontab option, and has not yet finished running.
+By default, ``./run_WE2E_tests.py`` will actively monitor jobs, printing to console when jobs are complete (either successfully or with a failure), and printing a summary file ``WE2E_summary_{datetime.now().strftime("%Y%m%d%H%M%S")}.txt``.
+However, if the user is using the legacy crontab option (by submitting ``./run_WE2E_tests.py`` with the ``--launch cron`` option), or if the user would like to summarize one or more experiments that either are not complete or were not handled by the WE2E test scripts, this status/summary file can be generated manually using ``WE2E_summary.py``.
```

Comment:
I think this flag is `--use_cron_to_relaunch`.

Reply:
It's getting updated with one of Mike's PRs (https://github.com/ufs-community/ufs-srweather-app/pull/871)! I suppose ideally, that PR should be listed as a dependency, but I was thinking that since it's referring to a deprecated capability anyway, it's probably fine. @MichaelLueken do you have any idea when Mike's PR will be merged? I'd rather remove this than wait indefinitely, since this doc update ticket is already from last sprint, and my next PRs depend on its merging. But it also seems like sometimes PRs come in and the documentation is updated subsequently; this is a reversal of that. I realize that the ideal is for them to be updated together, but it seems pretty minor in this case. Let me know what you'd like me to do.

Reply:
Unfortunately, I don't know when Mike's PR will be merged.  I have given my approval, but I will need an approval from either Eddie or Gerard before I'll be able to submit the automated Jenkins tests.  Since PR #871 is included with the list of required changes for the SRW release, I think it would be fine to move forward with the `run_WE2E_tests.py` options that will be available following the merge of that PR.

Reply:
Oh! Gotcha. Sorry, I haven't kept entirely up-to-date on the other PRs.